### ISSUE TITLE
[question] easy way to apply tags to multiple files at once?

### ISSUE THREAD

**Issue by chad-bennett** [vnotex/vnote#1137]
migrating old content into vnote and wondering if there is an easy way to apply tags to multiple files at once? seems tedious to go file by file to add tags if a system already exists to speed up the process.

**Comment by tamlok** [comment]
No. You could write Python script to edit the `_vnote.json` file.

**Comment by tamlok** [comment]
Will be supported in next version by selecting multiple files and click the Tags action.
